tools/nix/parts/all.nix

```diff
@@ -22,8 +22,11 @@
 
       shellHook = ''
         ${config.devShells.rust.shellHook or ""}
-        export HOME=$PWD/.home
         export DYLD_FALLBACK_LIBRARY_PATH=$(rustc --print sysroot)/lib
+
+        export HOME=$PWD/.home
```

Comment:
Please drop this line altogether as well - anyone who uses direnv and nix-direnv (as I do) will find that nearly all of the software on their system that uses XDG config dirs is now broken while inside this devShell. Most values of `$EDITOR`, for example.

@mrinalwadhwa do you recall what this was being set for? Typically we can relocate this env var to a wrapper script or other more localized definition to control for whatever it helps with.

Reply:
I'm resolving the conversation. Please open a PR when this one is merged